### PR TITLE
[19325] Bump to 2.11.2 and release notes

### PR DESCRIPTION
**Pull request by Mario-DL**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/118/)

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/119/)

File: docs/notes/previous_versions/v2.11.1.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Version 2.11.1
+==============
```
Comment (EduPonz) [inline docs/notes/previous_versions/v2.11.1.rst:2]: Please underline with `^`

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/120/)

**Review by EduPonz — APPROVED** [review]

**Review by EduPonz — APPROVED** [review]

**Pull request merged**